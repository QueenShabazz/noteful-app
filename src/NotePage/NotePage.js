import React from 'react'
import Note from '../Note/Note'

export default function NotePage(props){
    return(
        <section>
            <button
                tag='button'
                role='link'
                onClick={()=> props.history.goBack()}
                >
                    Back
            </button>
            {props.folder && (
                <h3>
                    {props.folder.name}
                </h3>
            )}
        </section>
    )
}

NotePage.defaultProps={
    history:{
        goBack: ()=>{}
    }
}
