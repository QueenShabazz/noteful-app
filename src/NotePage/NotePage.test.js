import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotePage from './NotePage'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() });

describe(`Form component`, () => {

  it('renders the Form default', () => {
    const wrapper = shallow(
        <NotePage />
        )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  
})