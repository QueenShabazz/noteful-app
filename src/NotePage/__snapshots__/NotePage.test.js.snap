// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component renders the Form default 1`] = `
<section>
  <button
    onClick={[Function]}
    role="link"
    tag="button"
  >
    Back
  </button>
</section>
`;
